app/javascript/modules/listingDetailsLottery/ListingDetailsLotteryResultsRow.tsx

```diff
@@ -23,13 +23,20 @@ export const ListingDetailsLotteryResultsRow = ({ bucket }: LotteryResultsRowPro
           {bucket.preferenceName === "generalLottery"
             ? t("lottery.generalPool")
             : defaultIfNotTranslated(
-                `listings.lotteryPreference.${bucket.preferenceName}.title`,
-                bucket.preferenceName
+                `listings.lotteryPreference.${bucket.preferenceName
+                  .replace("Tier 1 ", "")
+                  .replace("Tier 2 ", "")}.title`,
+                bucket.preferenceName.replace("Tier 1 ", "").replace("Tier 2 ", "")
               )}
         </Heading>
+        {bucket.preferenceName.includes("Tier 1")
+          ? t("listings.lotteryPreference.sfusd.educators.shortTitle")
+          : (bucket.preferenceName.includes("Tier 2")
+          ? t("listings.lotteryPreference.sfusd.allOtherEmployees.shortTitle")
+          : t("listings.lotteryPreference.sfusd.generalPublic.shortTitle"))}
         {bucket.preferenceName !== "generalLottery" && (
           <p className="mb-1 text-gray-950 text-sm">
-            {t("lottery.upToXUnitsAvailable", { units: bucket.unitsAvailable })}
+            {t("lottery.upToXUnitsAvailable", { units: bucket.unitsAvailable ?? 0 })}
```

Comment:
question: Have we discussed if this is the best way to handle blank values? I could be wrong, but blank values shouldn't happen in Production and usually indicate a data issue in Salesforce.

Reply:
We have not! I agree - won't happen in production

Is that a design question ie "If there are blank values, what should it look like"? Or we can just remove this change to line 39 and continue assuming there won't be blank values

Reply:
The current implementation is actually better for detecting data issues, even if it is ugly. We've caught things in the past because `up to %{units}` is really easy to spot.